aicsimageio/metadata/utils.py

```diff
@@ -422,3 +425,67 @@ def clean_ome_xml_for_known_issues(xml: str) -> str:
         )
 
     return xml
+
+
+def dtype_to_ome_type(npdtype: np.dtype) -> PixelType:
```

Comment:
shouldn't this be a static method? Then it can be used outside the class without having to instantiate the class.

Reply:
None of the methods in this module are scoped to any class, they're just regular exported functions.